src/main/java/Duke.java

```diff
@@ -76,25 +78,20 @@ public static void invalidTask() {
     }
 
     public static void addTask(String line) {
+        Task t;
         try {
             String[] commands = line.split(" ", 2);
             String type = commands[0];
             String description = commands[1];
-            Task t;
 
-            switch (type) {
-            case "todo":
+            if (type.equals("todo")) {
                 t = new Todo(description);
-                break;
-            case "deadline":
+            } else if (type.equals("deadline")) {
                 t = parseDeadline(description);
-                break;
-            case "event":
+            } else if (type.equals("event")) {
                 t = parseEvent(description);
-                break;
-            default:
-                // can set t to null because will catch this exception
-                t = null;
+            } else {
+                throw new java.lang.RuntimeException("Not a valid task type");
```

Comment:
did you have any particular reason why you switched from switch to if-else statements?